README.md

```diff
@@ -1 +1,57 @@
-# ci-script
\ No newline at end of file
+# ci-script
+
+Script to help setup a repo with continuous integration support.
+
+- Uses GitHub Actions.
+- Actions configs for different setups.
+
+## Usage
+
+From your source dir, run:
+
+    ../ci-script/setup-ci MODE
+
+Where MODE is one of:
+
+- bedrock
+- bedrock-web
+- isomorphic
+- lint
+
+## Lint
+
+If you are using lint support:
+
+    npm i -D eslint
+    npm i -D eslint-config-digitalbazaar
```

Comment:
```suggestion
    npm i -D eslint eslint-config-digitalbazaar
```

Also, if you're giving instructions for setting up linting in the first place, need to say something about eslintrc.js.

Reply:
I did say below to setup the .eslintrc.js.  How about deferring to eslint-config-digitalbazaar README.  This isn't the place to describe that setup.